File: AIResources/Python/testai.py
from hlt import *
from networking import *

sock = connectToGame()
playerTag, gameMap = getInit(sock)
sendInit(sock)

moves = []

while True:
	gameMap = getFrame(sock)
	for a in range(0, len(gameMap.contents)):
		for b in range(0, len(gameMap.contents[a])):
			site = gameMap.contents[a][b]
			if site.owner == playerTag:
				moves.append(Move(Location(b, a), 1))
	sendFrame(sock, moves)

File: AIResources/Java/TestAI.java
import java.net.*;
import java.util.ArrayList;

public class TestAI
{
    public static void main(String[] args) {
        Socket sock = Networking.connectToGame();

        InitPackage iPackage = Networking.getInit(sock);
        short playerTag = iPackage.playerTag;
        Map gameMap = iPackage.map;

        Networking.sendInit(sock);

        while(true) {
            ArrayList<Move> moves = new ArrayList<Move>();
            gameMap = Networking.getFrame(sock);

            for(int y = 0; y++ < gameMap.contents.size(); y++) {
                for(int x = 0; x < gameMap.contents.get(y).size(); x++) {
                    Site site = gameMap.contents.get(y).get(x);
                    if(site.owner == playerTag) {
                        moves.add(new Move(new Location((short)x, (short)y), Direction.EAST));
                        
                    }
                }
            }
            
            Networking.sendFrame(sock, moves);
        }
    }
}
